### PR TITLE
Feature/migration to fabric 1.3

### PR DESCRIPTION
**Pull request by fedyiva**
-migrated to fabric-utils 1.3.2
-changed fabric docker images to 1.3

### CODE REVIEW

File: bootstrap/bootStrap.sh [hunk 1]
```diff
@@ -463,25 +467,32 @@ echo "Configuring Instance (output: $configLog)"
...
 spinner $! 1
 
+#workaround for upgrading PyYaml
+ssh -q -i  "$keyFile" "${instanceUser}@${PublicDnsName}" \
+	"sudo  bash -c 'rm -rf /usr/lib/python2.7/dist-packages/yaml && rm -rf /usr/lib/python2.7/dist-packages/PyYAML-3.11.egg-info'" >> $configLog 2>&1 &
+
 #run pip install for the package
 ssh -q -i  "$keyFile" "${instanceUser}@${PublicDnsName}" \
-	"sudo -u root -H pip install -r \
+	". .bashrc && sudo -u root -H pip install -r \
```
Comment (osesov) [inline bootstrap/bootStrap.sh]: loading bashrc this way looks awful.
can we use "bash -c ..." or "sudo -i ..." instead?

File: ops/configurator/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,29 @@
-FROM frolvlad/alpine-oraclejdk8
-MAINTAINER Nikita Volkov
-COPY fabric-configurator-1.2.16.jar fabric-configurator-1.2.16.jar
+#steps to get OpenJDK8
+FROM alpine:3.9
```
Comment (osesov) [inline ops/configurator/Dockerfile]: ca we use "openjdk:8-alpine" instead of setting up java manually?

File: ops/configurator/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,29 @@
...
+#Workaround to avoid exeptions formn netty
+RUN ln -s  /lib/libcrypto.so.1.1 /lib/libcrypt.so.1
+#fabric-skeleton specific configuration
+
+COPY fabric-configurator-fat-1.3.2.jar fabric-configurator-fat-1.3.2.jar
 COPY network network
 COPY chaincodes chaincodes
 COPY fabric.yaml fabric.yaml
-CMD java -cp fabric-configurator-1.2.16.jar com.luxoft.fabric.config.Configurator --type config
+CMD java -cp fabric-configurator-fat-1.3.2.jar com.luxoft.fabric.config.Configurator --type config
```
Comment (osesov) [inline ops/configurator/Dockerfile]: Let's think of passing version from outside (using ARG or ENV)?

File: bootstrap/bootStrap.sh [hunk 4]
```diff
@@ -410,10 +414,10 @@ while [ "$accessible" -ne 0 ];do
 	sleep 5 &
 	#	pid=$!
 	spinner $! 1
-	accessible=$(ping -t1 -i1 -n -c 1 $PublicDnsName >/dev/null 2>&1;echo $?)
```
[the remark this answers is not in the document]
Comment (fedyiva) [reply bootstrap/bootStrap.sh]: I had the same question. That's why I removed it. Please pay attention to the fact that this part shows red, means it is removed

File: bootstrap/bootStrap.sh [hunk 5]
```diff
@@ -410,10 +414,10 @@ while [ "$accessible" -ne 0 ];do
...
+	accessible=$(ping -i1 -n -c 1 $PublicDnsName >/dev/null 2>&1;echo $?)
 	#Now make sure ssh is up
 	if [ "$accessible" -eq 0 ];then
-		accessible=$(ssh -q -i  "$keyFile" "${instanceUser}@$PublicDnsName" \
+		accessible=$(ssh -q -i  "$keyFile" -o StrictHostKeyChecking=no "${instanceUser}@$PublicDnsName" \
```
[the remark this answers is not in the document]
Comment (fedyiva) [reply bootstrap/bootStrap.sh]: My purpose is to add key to known hosts silently  on the first connection attempt to the new host

File: ops/configurator/Dockerfile (hunk 3 above)
Comment (fedyiva) [reply ops/configurator/Dockerfile]: I agree, moreover I  think we should not keep this jar in the source code repo, but download it from some binary repository instead.

This seems like improvement for me, while I think the first priority is to make the project working first and then polish it.

File: bootstrap/bootStrap.sh (hunk 1 above)
Comment (fedyiva) [reply bootstrap/bootStrap.sh]: Fixed

File: ops/configurator/Dockerfile (hunk 2 above)
Comment (fedyiva) [reply ops/configurator/Dockerfile]: Fixed

File: bootstrap/bootStrap.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (osesov) [reply bootstrap/bootStrap.sh]: ok, sorry

**Pull request merged**